### ISSUE TITLE
Don't overwrite existing ahoy.yml on init

### ISSUE THREAD

**Issue by bjeavons** [ahoy-cli/ahoy#4]
`init` should check for an existing .ahoy.yml file and prompt for overwrite before doing so.

**Comment by frankcarey** [comment]
This is a good idea, or at least have it check first, but you can actually update this yourself. If you implement an "init" command in .ahoy.yml, you can have it do (or not do) anything you want to.

Example: 

```Yaml
commands:
  init:
    cmd: echo "These aren't the droids you're looking for.."
```

**Comment by AlexSkrypnyk** [comment]
This issue is still present in Ahoy 2 - running `ahoy init` on the project with existing `.ahoy.yml` silently overrides it. Having a prompt would be helpful.

**Comment by ocean** [comment]
Closing as this is fixed with #97
